ArduCopter/heli.cpp

```diff
@@ -151,6 +151,18 @@ float Copter::get_pilot_desired_rotor_speed() const
     return 0.0f;
 }
 
+// get throttle from transmitter
+// returns -1 if throttle hasn't been defined
+float Copter::get_pilot_desired_throttle() const
+{
+    RC_Channel *rc_ptr = rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_INTERLOCK);
+    if (rc_ptr != nullptr) {
+        rc_ptr->set_range(1000);
+        return (float)rc_ptr->get_control_in() * 0.001f;
```

Comment:
I don't think the "(float)" is required.  The multiply by "0.001f" should ensure that the final value is a float.